get-latest.sh

```diff
@@ -36,11 +36,33 @@ if [ $? -ne 0 ]; then
 	exit 1;
 fi
 
-# Get a list of version numbers that are available
-cat /tmp/tc${tomcat_major_version}-index.html | fgrep '[DIR]' | grep -E "v${tomcat_major_version}.${tomcat_minor_version}.[0-9]+/" | grep -Eo '<a[ \t]+href[ \t]*=[ \t]*"[^"]*"' | grep -Eo "v${tomcat_major_version}\.${tomcat_minor_version}\.[^\"]*" | sed 's|/$||' > /tmp/tc${tomcat_major_version}-versions.txt
+# Attempt to get the Patch Version Number
+first_v=0
+first_dot=0
+second_dot=0
+cat /tmp/tc${tomcat_major_version}-index.html | grep -o [v"$tomcat_major_version"."$tomcat_minor_version".0-9] > /tmp/tc"$tomcat_major_version"-versions.txt
+tomcat_version_str=cat /tmp/tc"$tomcat_major_version"-versions.txt
+tomcat_patch_version=""
+
+for char in $tomcat_version_str; do
```

Comment:
Replacing this line with `for char in ${cat /tmp/tc"$tomcat_major_version"-versions.txt}; do` resulted in a new error.

get-latest.sh: line 51: ${cat /tmp/tc"$tomcat_major_version"-versions.txt}: bad substitution